src/event_loop.rs

```diff
@@ -33,19 +30,6 @@ impl EventLoopConfig {
         }
     }
 
-    /// Sets the default amount of time that a thread will be blocked in I/O
-    /// before timing out.
-    ///
-    /// If the event loop receives no I/O events during the specified timeout
-    /// then it will use this timeout to return control back to the original
-    /// program and run one more tick of the event loop.
-    ///
-    /// The default value for this is 1000.
-    pub fn io_poll_timeout_ms(&mut self, timeout: usize) -> &mut Self {
-        self.io_poll_timeout_ms = timeout;
-        self
-    }
```

Comment:
Perhaps the config could provide the ability to customize the `tick_ms` option as well?